spec/10-expressions.md

```diff
@@ -764,6 +764,11 @@ $obj2 = clone $obj1;  // creates a new Manager that is a deep copy
   <i>object-creation-expression:</i>
     new  <i>class-type-designator</i>  (  <i>argument-expression-list<sub>opt</sub></i>  )
     new  <i>class-type-designator</i>
+    new  class  (  <i>argument-expression-list<sub>opt</sub></i>  )
+        <i>class-base-clause<sub>opt</sub></i>   <i>class-interface-clause<sub>opt</sub></i>
+        {   <i>trait-use-clauses<sub>opt</sub></i>   <i>class-member-declarations<sub>opt</sub></i> }
+    new  class <i>class-base-clause<sub>opt</sub></i>  <i>class-interface-clause<sub>opt</sub></i>
+        {   <i>trait-use-clauses<sub>opt</sub></i>   <i>class-member-declarations<sub>opt</sub></i> }
```

Comment:
I just confirmed that is the case, so will fix the grammar in the Classes chapter as well as in the new operator.